recipes/psrqpy/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "psrqpy" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "47c550a05ccbe8e5b1217dcd9d51d46fc0a36d643395de2075c637ff610c7fb8"
```

Could you please remove the quotes from ``sha256`` ?
```suggestion
  sha256: 47c550a05ccbe8e5b1217dcd9d51d46fc0a36d643395de2075c637ff610c7fb8
```